#! /bin/sh /usr/share/dpatch/dpatch-run
## config-sbj.dpatch by  <jame@rocasa.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove the xtrn/sbj/sbj & xtrn/sbj/sbjclean files that come with
## DP: the source archive, as they are actually only scripts that attempt
## DP: to locate the real executables in a build directory, which is not
## DP: needed for the debian package.

@DPATCH@
diff -urNad synchronet_3.14a~/xtrn/sbj/sbj synchronet_3.14a/xtrn/sbj/sbj
--- synchronet_3.14a~/xtrn/sbj/sbj	2004-10-26 14:54:54.000000000 -0400
+++ synchronet_3.14a/xtrn/sbj/sbj	1969-12-31 19:00:00.000000000 -0500
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
diff -urNad synchronet_3.14a~/xtrn/sbj/sbjclean synchronet_3.14a/xtrn/sbj/sbjclean
--- synchronet_3.14a~/xtrn/sbj/sbjclean	2004-10-26 14:54:54.000000000 -0400
+++ synchronet_3.14a/xtrn/sbj/sbjclean	1969-12-31 19:00:00.000000000 -0500
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
